scripts/build.js

```diff
@@ -70,40 +65,43 @@ function getBuildPath(file, buildFolder) {
   return path.resolve(pkgBuildPath, relativeToSrcPath);
 }
 
-function buildPackage(p) {
+function buildNodePackage(p) {
   const srcDir = path.resolve(p, SRC_DIR);
   const pattern = path.resolve(srcDir, '**/*');
   const files = glob.sync(pattern, {nodir: true});
 
   process.stdout.write(fixedWidth(`${path.basename(p)}\n`));
 
   files.forEach(file => buildFile(file, true));
+
   process.stdout.write(`[  ${chalk.green('OK')}  ]\n`);
 }
 
-function buildFile(file, silent) {
-  buildFileFor(file, silent, 'node');
-
-  const pkgJsonPath = path.resolve(
-    PACKAGES_DIR,
-    getPackageName(file),
-    'package.json'
-  );
-  const {browser} = require(pkgJsonPath);
-  if (browser) {
-    if (browser.indexOf(BUILD_ES5_DIR) !== 0) {
-      throw new Error(
-        `browser field for ${pkgJsonPath} should start with "${BUILD_ES5_DIR}"`
+async function buildBrowserPackages(packages) {
```

Comment:
Using `async/await` and `for of` loops here in order for getting a nice `stdout` output similar to the node builds.